worker/paidAction.js

```diff
@@ -0,0 +1,133 @@
+import { paidActions } from '@/api/paidAction'
+import { datePivot } from '@/lib/time'
+import { Prisma } from '@prisma/client'
+import { getInvoice } from 'ln-service'
+
+async function transitionInvoice (jobName, { invoiceId, fromState, toState, toData, onTransition }, { models, lnd, boss }) {
+  console.group(`${jobName}: transitioning invoice ${invoiceId} from ${fromState} to ${toState}`)
```

Comment:
We should start using `console.group` more! It'd be especially useful clientside.